mcp_client/README.md

```diff
@@ -0,0 +1 @@
+# MCP Client
```

Comment:
these are linked in the pyproject.toml so i just added them as a placeholder for now

`readme = "README.md"`